experimental/devices/sn3218/example_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package sn3218_test
+
+import (
+	"log"
```

Comment:
Please use `goimports -w`. I recommend to setup your text editor to invoke it automatically upon file save.

http://golang.org/x/tools/cmd/goimports

Reply:
fixed, will have a look at goimports